Server/BackEndMonitoringServer/src/CContainerOfProcesses.cpp

```diff
@@ -0,0 +1,107 @@
+#include "stdafx.h"
+
+#include <windows.h>
+#include <psapi.h>
+#include <algorithm>
+
+#include "CContainerOfProcesses.h"
+
+CContainerOfProcesses::CContainerOfProcesses(unsigned m_max_process_count,
+											std::chrono::duration<int> 
```

Comment:
Something wrong with white spaces

Reply:
weird. On my pc it's look ok. Ok, I out params right under constructor

Reply:
Fixed, [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/98be71169fd1e86ad17729bdfd628a367b109dc7)